projects/noloan/admin/src/main/java/hasadna/noloan/admin/app/SuggetionRecyclerAdapter.java

```diff
@@ -81,15 +82,13 @@ public void bind(SmsMessage sms) {
       receivedAt.setText(sms.getReceivedAt());
       buttonAccept.setOnClickListener(
           view -> {
-            FirestoreClient client = new FirestoreClient();
-            client.writeMessage(sms);
```

Comment:
Why did we add and then delete?

Reply:
We added to the spam list and deleted from the suggested list. 
Told you admin is not working for a while now